ceph.spec.in

```diff
@@ -809,15 +809,13 @@ mkdir -p %{_localstatedir}/run/ceph/
 %{_mandir}/man8/ceph-clsinfo.8*
 %{_mandir}/man8/librados-config.8*
 #set up placeholder directories
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/tmp
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/mon
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/osd
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/mds
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-osd
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-mds
-%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-rgw
-%attr(770,ceph,ceph) %dir %{_localstatedir}/run/ceph/
```

Comment:
the sysvinit and upstart scripts create it on demand (debian doesn't 
create it on package install at all)

On Thu, 10 Sep 2015, branto1 wrote:

> 
> In ceph.spec.in:
> 
> > @@ -809,15 +809,13 @@ mkdir -p %{_localstatedir}/run/ceph/
> >  %{_mandir}/man8/ceph-clsinfo.8*
> >  %{_mandir}/man8/librados-config.8*
> >  #set up placeholder directories
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/tmp
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/mon
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/osd
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/mds
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-osd
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-mds
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/lib/ceph/bootstrap-rgw
> > -%attr(770,ceph,ceph) %dir %{_localstatedir}/run/ceph/
> 
> Here, you removed /var/run/ceph but did not place it anywhere else. While this might be ok for systemd builds, I suppose
> it might cause problems for non-systemd builds (and systemd is a conditional build flag).
> 
> ?
> Reply to this email directly or view it on GitHub.[AAabh8UKWm8iemzTkDiOkFI1iqn7aIXlks5owYLzgaJpZM4F6847.gif]
> 
> 
>